dss/api/bundles/__init__.py

```diff
@@ -65,8 +67,28 @@ def get(
             headers['Retry-After'] = RETRY_AFTER_INTERVAL
             return response
 
+    files = bundle_metadata[BundleMetadata.FILES]
+    if start_at > 0:
+        files = files[start_at:]
```

Comment:
nit: it might be better to slice `files` just once, i.e., `files[start_at:start_at + per_page]`

Reply:
Looks cleaner that way. Fixed.